src/main/java/org/sonarlint/intellij/core/ProjectBindingManager.java

```diff
@@ -47,6 +56,36 @@ public ProjectBindingManager(Project project, SonarLintEngineManager engineManag
     this.console = console;
   }
 
+  @Override
+  public void initComponent() {
+    JobScheduler.getScheduler().scheduleWithFixedDelay(new Runnable() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this anonymous class number of lines from 26 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1188)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this anonymous inner class a lambda [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1604)